trunk/SUAVE/Methods/Flight_Dynamics/Static_Stability/compute_aero_derivatives.py

```diff
@@ -12,7 +12,7 @@
 
 ## @ingroup Methods-Flight_Dynamics-Static_Stability
 def compute_aero_derivatives(segment, h=1e-4): 
-    """This function takes an computes the aerodynamic derivatives about a segment.
+    """This function computes the aerodynamic derivatives about a segment.
     
     Assumptions:
        Linearized equations are used for each state variable
```

Comment:
I actually would like this updated. These docstrings [generate our documentation](https://suave.stanford.edu/doxygen/). This could be an important script for a user, they need to know what this does. They need to know what sort of inputs and assumptions are baked in. They especially need to know what derivates are calculated.

Reply:
Also how does one tweak h in a mission?

Reply:
updated and moved h to the main script instead